include/plog/Log.h

```diff
@@ -24,7 +24,7 @@
 #   define PLOG_GET_FUNC()      __PRETTY_FUNCTION__
 #endif
 
-#if PLOG_CAPTURE_FILE
+#ifdef PLOG_CAPTURE_FILE

```

Comment:
Is there any problem with `#if` instead of `#ifdef` on RTEMS? I compiled fine without thease changes (rtems4.11/gcc4.9.3).

Reply:
It is probably because we compile with a quite pedantic compiler setting. I got a "-Wundef" warning during compilation. As far as I see it by default the `PLOG_CAPTURE_FILE` is not defined at all, so I thought ifdef would be the better choice here.

Reply:
Good point, I'll add `-Wundef` to the flags.